src/tokens/TabWallet.tsx

```diff
@@ -1,231 +1,82 @@
-  }
-
-  const handleMeasureListHeaderHeight = (event: LayoutChangeEvent) => {
-    setListHeaderHeight(event.nativeEvent.layout.height)
-  }
-
-  const handleMeasureListFooterHeight = (event: LayoutChangeEvent) => {
-    setListFooterHeight(event.nativeEvent.layout.height)
-  }
-
-  const handleChangeActiveView = (selectedTab: AssetTabType) => {
-    navigation.setParams({ activeAssetTab: selectedTab })
-  }
-
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import { useTotalTokenBalance } from 'src/tokens/hooks'
+import { cKESFirstTokensListSelector } from 'src/tokens/selectors'
+import { TokenBalanceItem } from 'src/tokens/TokenBalanceItem'
+import { getSupportedNetworkIdsForTokenBalances, getTokenAnalyticsProps } from 'src/tokens/utils'
+
+function TabWallet() {
+  const hideWalletBalances = useSelector(hideWalletBalancesSelector)
+  const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
+  const { decimalSeparator } = getNumberFormatSettings()
+
+  const supportedNetworkIds = getSupportedNetworkIdsForTokenBalances()
+  const tokens = useSelector((state) => cKESFirstTokensListSelector(state, supportedNetworkIds))
+  const totalTokenBalanceLocal = useTotalTokenBalance()
+  const balanceDisplay = hideWalletBalances
+    ? `XX${decimalSeparator}XX`
+    : totalTokenBalanceLocal?.toFormat(2)
   return (
-    // Transparency issue on Android present when a fragment is used - Nested Animated.View prevents it
-    <Animated.View>
-      <Animated.View
-        style={[styles.listHeaderContainer, animatedListHeaderStyles]}
-        onLayout={handleMeasureListHeaderHeight}
-      >
-        <View
-          style={[styles.nonStickyHeaderContainer]}
-          onLayout={handleMeasureNonStickyHeaderHeight}
-        >
-          <View style={styles.balanceSection}>
-            <AssetsTokenBalance showInfo={false} />
-          </View>
-          <ActionsCarousel key={'ActionsCarousel'} />
-        </View>
-        <AssetTabBar
-          activeTab={activeTab}
-          onChange={handleChangeActiveView}
-          displayPositions={displayPositions}
-        />
-      </Animated.View>
-
-      <AssetList
-        activeTab={activeTab}
-        listHeaderHeight={listHeaderHeight}
-        handleScroll={handleScroll}
-      />
-      {showClaimRewards && (
-        <Animated.View
-          style={[styles.footerContainer, animatedFooterStyles]}
-          onLayout={handleMeasureListFooterHeight}
-        >
-          <Button
-            type={BtnTypes.SECONDARY}
-            size={BtnSizes.FULL}
-            text={t('assets.claimRewards')}
+    <SafeAreaView style={styles.container} edges={['top']}>
+      <View style={styles.row}>
+        <Text style={styles.totalBalance} testID={'TotalTokenBalance'}>
+          {!hideWalletBalances && localCurrencySymbol}
+          {balanceDisplay}
+        </Text>
+        <HideBalanceButton hideBalance={hideWalletBalances} />
+      </View>
+      <ScrollView>
```

Comment:
```suggestion
      <ScrollView contentContainerStyle={{ marginTop: Spacing.Large32}}>
```
Add marginTop to this ScrollView and remove the `marginBottom: Spacing.Large32` from the row styles the tokens will disappear behind the numbers instead of the whitespace used in the margin on the numbers.